README.md

```diff
@@ -617,6 +617,10 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 | :green_heart:  | STM32L041G6 | Generic Board | *2.4.0* |  |
 | :green_heart: | STM32L041K6T | Generic Board | *2.0.0* |  |
 | :green_heart: | STM32L051C6<br>STM32L051C8 | Generic Board | *2.0.0* |  |
+| :yellow_heart: | STM32L051K6T | Generic Board | **2.8.0** |  |
+| :yellow_heart: | STM32L051K8T | Generic Board | **2.8.0** |  |
+| :yellow_heart: | STM32L051K6U | Generic Board | **2.8.0** |  |
+| :yellow_heart: | STM32L051K8U | Generic Board | **2.8.0** |  |
```

Comment:
Can be simplified as it references all version (only T and U).
```suggestion
| :yellow_heart: | STM32L051K6<br>STM32L051K8 | Generic Board | **2.8.0** |  |
```

Reply:
Done.